official/resnet/keras/keras_imagenet_benchmark.py

```diff
@@ -510,7 +629,24 @@ def benchmark_xla_8_gpu_fp16_tensorboard_tweaked(self):
         'benchmark_xla_8_gpu_fp16_tensorboard_tweaked')
     FLAGS.batch_size = 256 * 8  # 8 GPUs
     FLAGS.tf_gpu_thread_mode = 'gpu_private'
-    FLAGS.data_prefetch_with_slack = True
+    FLAGS.data_delay_prefetch = True
+    FLAGS.enable_tensorboard = True
+    self._run_and_report_benchmark()
+
+  def benchmark_xla_8_gpu_fp16_tensorboard_tfdata_exp(self):
```

Comment:
I think one test for tracking tensorboard overhead is enough so this might not be necessary.

Reply:
Removed, thanks